pennylane/tape/tapes/reversible.py

```diff
@@ -181,7 +181,7 @@ def analytic_pd(self, idx, device, params=None, **options):
 
         generator = generator(wires)
 
-        diff_circuit = QuantumTape()
+        diff_circuit = JacobianTape()

```

Comment:
I think this one can remain a `QuantumTape`

Reply:
Cool. I don't understand the details, but makes sense, since we are inside the jacobian method.

Reply:
Yep, it's since the `diff_circuit` doesn't need to be differentiated, just executed. So we probably save some overhead not making it a Jacobian tape.